CS3230Project/View/SearchPatient.cs

```diff
@@ -65,7 +65,7 @@ private void displayPatientData(List<Patient> patientsToDisplay)
                 string[] patientDetails =
                 {
                     currPatient.PatientId.ToString(), currPatient.LastName, currPatient.FirstName,
-                    currPatient.DateOfBirth.ToString(), currPatient.Gender, currPatient.PhoneNumber,
+                    currPatient.DateOfBirth.ToString("d"), currPatient.Gender, currPatient.PhoneNumber,
```

Comment:
What exactly is this "d" here?

Reply:
it is for the date format. I will check into it and see if it is needed or incorrect format.

Reply:
I changed it to "MM/dd/yyyy", but "d" does the exact same thing. I suppose it is more readable with the actual format.